plugins/com.google.cloud.tools.eclipse.googleapis/.classpath

```diff
@@ -10,6 +10,7 @@
 	<classpathentry exported="true" kind="lib" path="lib/google-api-services-compute-v1-rev160-1.23.0.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/google-api-services-dataflow-v1b3-rev219-1.23.0.jar"/>
 	<classpathentry exported="true" kind="lib" path="lib/google-api-services-datastore-v1-rev34-1.23.0.jar"/>
+	<classpathentry exported="true" kind="lib" path="lib/google-api-services-iam-v1-rev222-1.23.0.jar"/>
```

Comment:
Did you move this?  Just wondering if there was a reason.

Reply:
Yeah, I noticed there are in an alphabetical order.